install_config/install/deploy_router.adoc

```diff
@@ -269,9 +280,63 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be grouped to distribute routing load in the cluster and separate
```

Comment:
How about:

An administrator can create multiple routers with the same definition that will all serve the same set of routes.  By having different groups of routers with different namespace or route selectors, they can vary the routes that the router will serve.